packages/nx/src/native/glob/glob_transform.rs

```diff
@@ -107,6 +113,33 @@ fn build_segment(
     }
 }
 
+pub fn partition_glob(glob: &str) -> (String, Vec<String>) {
+    let (negated, groups) = parse_glob(glob).unwrap();
```

Comment:
Make this a `Result` and use `?` here.
```suggestion
    let (negated, groups) = parse_glob(glob)?;
```